api/v1alpha4/machinehealthcheck_webhook.go

```diff
@@ -138,18 +137,11 @@ func (m *MachineHealthCheck) validate(old *MachineHealthCheck) error {
 	}
 
 	if m.Spec.MaxUnhealthy != nil {
-		if _, err := intstr.GetValueFromIntOrPercent(m.Spec.MaxUnhealthy, 0, false); err != nil {
+		if _, err := intstr.GetScaledValueFromIntOrPercent(m.Spec.MaxUnhealthy, 0, false); err != nil {
 			allErrs = append(
 				allErrs,
 				field.Invalid(field.NewPath("spec", "maxUnhealthy"), m.Spec.MaxUnhealthy, "must be either an int or a percentage"),
```

Comment:
Maybe we want to make the err text consistent to the one returned in the MachineDeployment webhook